admin_manual/installation/installation_wizard.rst

```diff
@@ -140,50 +140,210 @@ use :ref:`label-phpinfo` (Look for the **User/Group** line).
+         )
+ } || {
+     id -u apache   &>/dev/null && \
+         config=(
+             [distro]="Fedora / CentOS based" \
+             [htuser]=apache \
+             [htgroup]=apache \
+         )
+ } || {
+     id -u http     &>/dev/null && \
+         config=(
+             [distro]="Archlinux based" \
+             [htuser]=http \
+             [htgroup]=http \
+         )
+ } || {
+     id -u wwwrun   &>/dev/null && \
+         config=(
+             [distro]="OpenSUSE based" \
+             [htuser]=wwwrun \
+             [htgroup]=www \
+         )
+ } || { error_exit "unknown platform"; }
+
+ echo -e "\nDistro: ${config[distro]}\n"
+
+ id -u ${config[htuser]} &>/dev/null || \
+     error_exit "htuser: ${config[htuser]} does not exist" 2
+
+ id -g ${config[htgroup]} &>/dev/null || \
+     error_exit "htgroup: ${config[htgroup]} does not exist" 3
+
+ # Are we preparing for an upgrade?
+ #  - upgrade permissions need to be wider open
+ [ "${1}" = "upgrade" ] && { upgrade=1; } || { upgrade=0; }
+
+
+ # Determine Nextcloud installation, unless already set
+ [ -z "${nc_path}" ] && {
+     [ -f $(pwd)/occ ] && {
+         nc_path=$(pwd)
+     } || {
+         nc_path="${DefaultDocumentRoot}"
+         echo "Default Nextcloud path set"
+     }
+ }
+ [ -f "${nc_path}"/occ ] || error_exit "Missing Nextcloud occ file"
+
+ # Determine Nextcloud data path, unless already set
+ [ -z "${nc_data}" ] && {
+     nc_data=''
+     [ -d $(pwd)/data  ] && { nc_data=$(pwd)/data; }
+     [ -d $(pwd)--data ] && { nc_data=$(pwd)--data; }
+ }
+
+ [ -z "${nc_data}" ] && {
+     nc_data="${DefaultDocumentRoot}/data"
+     echo "Default data path set"
+ }
+
+ echo
+
+
+ # Report detils
+ # - exit if required path(s) do not exist
+ [ -d "${nc_path}" ] && {
+     echo -e "    Nextcloud installation path:   \"${nc_path}\""
+ } || {
+     error_exit  "     Missing Nextcloud directory -- "${nc_path}"\n"
+ }
+
+ [ -d "${nc_data}" ] && {
+     echo -e "            Nextcloud Data path:   \"${nc_data}\"\n"
+ } || {
+     error_exit "Missing Nextcloud data directory -- "${nc_data}"\n"
+ }
+
+ echo -e "                 htuser:htgroup   ${config[htuser]}:${config[htgroup]}\n"
+
+ # Allow for confirmation of details, if not suitable,
+ # then script can be interupted using <ctrl>-c
+ echo -en 'Hit <enter> to continue... or <ctrl>-c to quit '
+ read dummy
+
+ # If preparing for upgrade works, then set permissions more open, then exit
+ [ ${upgrade} -eq 1 ] && {
+     do_cmd chown -R ${config[htuser]}:${config[htgroup]} \
+         "${nc_path}" "${nc_data}"
+     echo -e "Ready for upgrade\n"
+     exit
+ }
+
+ echo "Creating possible missing Directories"
+ do_cmd mkdir -p "${nc_path}"/{assets,updater} "${nc_data}"
+
+ echo "chmod Files and Directories"
+ do_cmd find "${nc_path}"/ "${nc_data}"/ -type f -exec chmod 0640 {} \;
+ do_cmd find "${nc_path}"/ "${nc_data}"/ -type d -exec chmod 0750 {} \;
+
+ echo "chown Directories"
+ do_cmd chown -R 0:${config[htgroup]} "${nc_path}"/
```

Comment:
Using `0` as `root` user, this will always work and it will **always** be root anyway.